wrapper/wrapper.go

```diff
@@ -16,19 +16,30 @@ import (
 	"github.com/domsolutions/gopayloader/pkgs/payloader"
 )
 
-// Converts jwtCustomClaimsJSON from string to map[string]string
-func JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON string) map[string]string {
-	jwtCustomClaimsMap := map[string]string{}
+// Converts jwtCustomClaimsJSON from string to map[string]interface{}
+func JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON string) (map[string]interface{}, error) {
+	if jwtCustomClaimsJSON == "" {
+		return nil, nil
+	}
+
+	jwtCustomClaimsMap := map[string]interface{}{}
+
+	err := json.Unmarshal([]byte(jwtCustomClaimsJSON), &jwtCustomClaimsMap)
+	if err != nil {
+		return nil, err
+	}
 
-  json.Unmarshal([]byte(jwtCustomClaimsJSON), &jwtCustomClaimsMap)
-  return jwtCustomClaimsMap
+	return jwtCustomClaimsMap, nil
 }
 
 func RunGoPayLoader(reqURI, mTLScert, mTLSKey string, disableKeepAlive bool, reqs int64, conns uint, totalTime time.Duration, skipVerify bool, readTimeout, writeTimeout time.Duration, method string, verbose bool, ticker time.Duration, jwtKID, jwtKey, jwtSub, jwtCustomClaimsJSON, jwtIss, jwtAud, jwtHeader string, headers []string, body, bodyFile string, client string) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	jwtCustomClaimsMap := JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON)
+	jwtCustomClaimsMap, err := JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON)
```

Comment:
Do you mind putting this call within the config `Validate()` method as that's where we validate all flags passed. Also maybe put some context arround the error so user understands which flag has the error i.e.

```golang
fmt.Errorf("config: failed to parse custom json, got error; %v", err)
```
Other than that, think it all looks good and I'll approve and merge

Reply:
Thanks, got a build error with your changes, can you fix? https://github.com/domsolutions/gopayloader/actions/runs/5413001919/jobs/9839238805?pr=3

Reply:
https://github.com/domsolutions/gopayloader/actions/runs/5414583349/jobs/9841953843?pr=3 unit test failures

Reply:
Weird the tests are passing for me on Windows

![image](https://github.com/domsolutions/gopayloader/assets/10351705/6056133a-7b84-4c08-90c3-7205decb2f76)

![image](https://github.com/domsolutions/gopayloader/assets/10351705/245c1500-0452-4a84-9611-fea147c9d7a0)

Is there a way to see which files get generated on your side?

Reply:
I think it might have to do with a difference in file line endings between the OSes, because the tests fail on windows for the original tests, probably \r vs \r\n

Reply:
Latest commit 30fa3ba seems to have solved the issue, the filenames generated for JWTs is now the same regardless of line endings in the private key files